runtime/onert/core/src/util/ChromeTracingEventWriter.cc

```diff
@@ -115,6 +122,45 @@ std::string object(const CounterEvent &evt)
   return ::object(content);
 }
 
+std::string getSessionLabel(const DurationEvent &evt)
+{
+  return "$" + std::to_string(evt.session_index) + " sess";
+}
+
+std::string getSubgLabel(const DurationEvent &evt)
+{
+  return "$" + std::to_string(evt.subg_index) + " subg";
+}
+
+std::string getOpLabel(const OpDurationEvent &evt)
+{
+  if (evt.op_seq_size > 1)
+    return "$" + std::to_string(evt.op_index) + " " + evt.op_name + " (" +
+           std::to_string(evt.op_seq_size) + ")";
+  else
+    return "$" + std::to_string(evt.op_index) + " " + evt.op_name;
+}
+
```

Comment:
`LabelMaker` makes name of op specific for ChromeTracingEventWriter.
`TidMaker` makes `tid` row of for ChromeTracing screen.